package.json

```diff
@@ -10,7 +10,7 @@
     "README.md"
   ],
   "engines": {
-    "node": ">=16.13.0"
+    "node": ">=18.19.1"
```

Comment:
@thgreasi any reason why you picked 18.19.1? cli is currently on ^18.6.0

Reply:
Oh that's a good point. It seems that I just picked the latest I had locally given that I was doing a bump, but forgot to check the CLI. Nice catch :+1:

Reply:
Oh wait @otaviojacobi , the CLI is on `^20.6.0`. That's what you meant right?

Reply:
fwiw, afair I chose v18 b/c that was the minimum that supports native fetch w/o flags.

Reply:
:man_facepalming: I did misread it and though we were already bumping this to 20. Please disregard my comment